drivers/include/disp_dev.h

```diff
@@ -133,11 +143,27 @@ uint8_t disp_dev_color_depth(disp_dev_t *dev);
 /**
  * @brief   Invert the display device colors
  *
- * @param[in] dev       Network device descriptor
+ * @param[in] dev       Pointer to the display device
```

Comment:
:laughing:

Reply:
Now you know where the inspiration comes from ;)